src/textual/_animator.py

```diff
@@ -93,9 +95,19 @@ class SimpleAnimation(Animation):
     final_value: object
     easing: EasingFunction
     on_complete: CallbackType | None = None
+    animate_on_level: AnimationsEnum = AnimationsEnum.BASIC
```

Comment:
Not keen on this name. It doesn't read well to me. Given from the context it is clearly related to animation, can it not simply be "level" ?

Reply:
Sure.